<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WithdrawWindow</class>
 <widget class="QDialog" name="WithdrawWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(66, 66, 66);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="headerLabel">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 200, 221);
font: 63 12pt &quot;Lato Semibold&quot;;</string>
           </property>
           <property name="text">
            <string>Withdraw</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="typeLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 200, 221);
font: 63 10pt &quot;Lato Semibold&quot;;</string>
           </property>
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exitButton">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
max-width: 4em;
padding: 5px;</string>
           </property>
           <property name="text">
            <string>exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 200, 221);
font: 63 10pt &quot;Lato Semibold&quot;;</string>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="accountNumberLabel">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 200, 221);
font: 63 10pt &quot;Lato Semibold&quot;;</string>
           </property>
           <property name="text">
            <string>Account Number:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>63</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="balanceLabel">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 200, 221);
font: 63 10pt &quot;Lato Semibold&quot;;</string>
         </property>
         <property name="text">
          <string>Balance</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 200, 221);
font: 63 10pt &quot;Lato Semibold&quot;;</string>
           </property>
           <property name="text">
            <string>Amount (€):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="amountLine">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 200, 221);
border-color: rgb(255, 200, 221);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QPushButton" name="tenButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
max-width: 4em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>10 €</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="fourtyButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
max-width: 4em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>40 €</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="sixtyButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>60 €</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="hundredButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>100 €</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="twoHundredButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>200 €</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="fiveHundredButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>500 €</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="3" column="2">
              <widget class="QPushButton" name="enterButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>enter</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="oneButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
max-width: 3em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="fourButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>4</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="eightButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>8</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="sixButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>6</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="threeButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
max-width: 3em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>3</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="zeroButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="fiveButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>5</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="twoButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
max-width: 3em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>2</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="sevenButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>7</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QPushButton" name="nineButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>9</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="clearButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 63 9pt &quot;Lato Semibold&quot;;
color: rgb(66, 66, 66);
background-color:rgb(255, 200, 221);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color:rgb(255, 200, 221);
min-width: 1em;
padding: 5px;</string>
               </property>
               <property name="text">
                <string>clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
